pom.xml

```diff
@@ -46,7 +46,7 @@
     </parent>
 
     <properties>
-        <openig.version>2024.6.0</openig.version>
+        <openig.version>IG_VERSION_REPLACE</openig.version>
```

Comment:
Properties can be overridden on the command line, so you can specify the value of openig.version for the build to use: 
```
mvn clean install -Dopenig.version=2023.11.0
```

We want the pom.xml to document the version that we have built and tested the code with. Pipeline which override this are great to give us confidence to then do the upgrade on main etc but we can't have placeholders in the pom.xml as you can no longer simply build the project using standard mvn commands.